SQL Fundamentals/EmployeeInfoView.sql

```diff
@@ -0,0 +1,11 @@
+﻿CREATE VIEW [dbo].[EmployeeInfo]
```

Comment:
Rename view following naming convention rules (I shared the link in comment under trigger).